tools/capput/capput.c

```diff
@@ -157,6 +159,77 @@ pdel(void *arg)
     hse_kvdb_txn_free(targ->kvdb, txn);
 }
 
+void
+del_tombs(void *arg)
+{
+    struct thread_arg *targ = arg;
+    struct hse_kvdb_txn *txn;
+    uint64_t suffix = 1;
+    hse_err_t err;
+
+    txn = hse_kvdb_txn_alloc(targ->kvdb);
+    if (!txn)
+        fatal(ENOMEM, "Failed to allocate resources for txn");
+
+    pthread_setname_np(pthread_self(), __func__);
+
+    while (!killthreads) {
+        char key[sizeof(uint64_t) * 2];
+        uint64_t *p; /* prefix */
```

Comment:
Just call the variables prefix and suffix and then remove the comment :)

Reply:
Copied from the other function.  The presence of globals named "pfx" and "sfx" and similar cause a lot of confusion and I suspect is what drove the use of p and s.  Feel free to create a ticket and rename everything, I think it an exercise of diminishing return at this point...

Reply:
Hahaha!  Sigh...